SweetTweetApp/pom.xml

```diff
@@ -42,13 +42,6 @@
       <version>1.1.0</version>
     </dependency>
 
-    <dependency>
```

Comment:
I'll assume that dependency is unused and Stepa will not punish me for not asking him)